packages/orbit-components/src/ServiceLogo/ServiceLogo.ct.tsx

```diff
@@ -2,12 +2,27 @@ import * as React from "react";
 import { test, expect } from "@playwright/experimental-ct-react";
 
 import ServiceLogoStory from "./ServiceLogo.ct-story";
+import { NAME_OPTIONS, SIZE_OPTIONS } from "./consts";
+
+const GRAYSCALE_OPTIONS_SIZE = 2;
 
 test.describe("visual ServiceLogo", () => {
   test("default", async ({ mount, page }) => {
     const component = await mount(<ServiceLogoStory />);
     await page.waitForRequest(/images.kiwi.com/);
 
-    await expect(component).toHaveScreenshot();
+    const selector = `img:nth-of-type(${
+      Object.values(NAME_OPTIONS).length +
+      Object.values(SIZE_OPTIONS).length +
+      GRAYSCALE_OPTIONS_SIZE
+    })`;
+
+    await page.waitForSelector(selector);
+
+    await expect(component).toHaveScreenshot({
+      threshold: 0.3,
+      maxDiffPixelRatio: 0.02,
```

Comment:
These values seem a bit arbitrary, is there a specific reason behind them? Perhaps we can add a comment.

Reply:
I confess they were a bit trial and error, so not science at all. Since we're comparing images I just tried the minimum values that would allow the tests to pass, trying multiple times until achieving consistent results